src/Qir/Execution/Tests.Microsoft.Quantum.Qir.Tools/Util.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.IO;
+
+namespace Tests.Microsoft.Quantum.Qir.Tools
+{
+    public static class Util
+    {
+        public static void DeleteDirectory(DirectoryInfo directory)
```

Comment:
If it's OK to remove directory, all files and subdirectories recursively, consider public static void Delete (string path, bool recursive). It will result in a different behavior than current code: if subdirectories are present, current code will fail, but proposed code will delete them. Not sure if this is a significant difference in our case.

Reply:
Remove the `DeleteDirectory` function and replaced its usages with `directory.Delete(true)`.